src/components/checkboxOptions/checkboxOptions.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
```

Comment:
I think it is better to store this component inside the `pattern` folder, in pages, because it is used only for patterns. Is it right?

Reply:
File and folder should be called `checkbox-options`

Reply:
fixed